src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -64,12 +64,6 @@ <h2>{{ addon_name }}</h2>
               </td>
             </tr>
           {% endif %}
-          {% if version and version.compatible_apps[APP] %}
```

Comment:
we can leave this in an replace with either `version.compatible_apps[amo.FIREFOX]` or a loop that shows all `version.compatible_apps`

Reply:
I did that initially but the value seems dubious at best, and I ran into problems because it's also used as an include in abuse admin... I ran out of motivation to shave yet another yak...